client/components/Editor/styles/main.scss

```diff
@@ -36,6 +36,7 @@ h6 {
 .footnote {
 	vertical-align: super;
 	font-size: 0.85em;
+	line-height: 1;
```

Comment:
I didn't know you could do, this — I've been in the unfortunate habit of repeating the same constant between `font-size` and `line-height` for this purpose.

Reply:
neither did I! A user suggested it!